engine-tests/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+mod prelude {
+    pub use aurora_engine_types::types::*;
+    pub use aurora_engine_types::*;
+}
```

Comment:
I still don't like this `prelude` importing all the things pattern.

Previously we had a prelude to handle std vs no-std imports more cleanly. That made sense to me. However I do not see the value in this sort of prelude, and indeed see it as a potential source of confusion when there are multiple ways to import the same thing.